scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/descriptor/JobDescriptorImpl.java

```diff
@@ -240,6 +240,17 @@ public void reStart(TaskId taskId) {
         eligibleTasks.put(taskId, (EligibleTaskDescriptor) runningTasks.remove(taskId));
     }
 
+    synchronized public void restartAllRunningTasls() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)